app/src/ui/app.tsx

```diff
@@ -1323,6 +1324,18 @@ export class App extends React.Component<IAppProps, IAppState> {
             onDismissed={this.onPopupDismissed}
           />
         )
+      case PopupType.OversizedFiles:
+        return (
+          <OversizedFiles
+            fileNames={popup.fileList}
+            onDismissed={this.onPopupDismissed}
+            dispatcher={popup.dispatcher}
```

Comment:
This will become `dispatcher={this.props.dispatcher}` once you've removed `popup.dispatcher`.